cli/Cargo.toml

```diff
@@ -30,6 +30,9 @@ features = ["derive"]
 [dependencies.colored]
 version = "2"
 
+[dependencies.nix]
```

Comment:
We should cfg guard this dependency

Reply:
good catch! I've just done it

just a note, as all checks have passed then maybe it would be good to add windows build to the ci?